### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
Have made some changes to improve docs
### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -127,7 +126,7 @@ https://github.com/Kong/docs.konghq.com/
 ### Proposing a new plugin
 
 We **do not** accept new plugins into the core repository. The plugins that are
-currently part of this repository are there because of historical reasons, but
+currently part of this repository is there because of historical reasons, but
```
Comment (levfishbluefish) [inline CONTRIBUTING.md]: Incorrect change

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -176,7 +175,7 @@ to verify a few things:
   development documentation for additional details)
 - The tests are passing: run `make test`, `make test-all`, or whichever is
   appropriate for your change
-- Do not update CHANGELOG.md yourself. Your change will be included there in
+- Do not update CHANGELOG.md yourself. Your change will be included therein
```
Comment (levfishbluefish) [inline CONTRIBUTING.md]:
```suggestion
- Do not update CHANGELOG.md yourself. Your change will be included in
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -237,7 +236,7 @@ commit history? Will they find unrelated changes in a particular commit? They
 shouldn't!
 
 Writing meaningful commit messages that follow our commit message format will
-also help you respect this mantra (see the below section).
+also, help you respect this mantra (see the below section).
```
Comment (levfishbluefish) [inline CONTRIBUTING.md]: incorrect change

**Comment by guanlan** [comment]
Thanks for the PR.

**Pull request closed without merging**